documentation/tutorials/get-started.md

```diff
@@ -46,6 +47,10 @@ Let's fetch everything:
 $ mix deps.get
 ```
 
+> ### Picosat installation issues? {: .info}
+>
+> Replace `{:picosat_elixir, "~> x.x"}` with `{:simple_sat, "~> x.x"}` to use a simpler (but mildly slower) solver. You can always switch back to `picosat_elixir` later once you're done with the tutorial.
```

Comment:
I'd add something along the lines of "if you have trouble compiling `picosat_elixir`...". We want folks to start there and only use `simple_sat` if they have issues with installation.

Reply:
OK will do

Reply:
Done